mail_activity_plan/static/src/js/mail_activity_plan.js

```diff
@@ -0,0 +1,31 @@
+odoo.define("mail_activity_plan.mail_activity_plan", function (require) {
+    "use strict";
+
+    const ListController = require("web.ListController");
+    var core = require("web.core");
+    var _t = core._t;
+
+    ListController.include({
+        async _actionWizardMailActivityPlan() {
```

Comment:
Can you avoid to show the entry if no existing plans?

Reply:
You can do it in the `_getActionMenuItems` function with an rpc call to a simple search.

Reply:
You are right, `_getActionMenuItems()` could be used to make an rpc call, but it is very important to clarify that this method is called **every time** a record is selected / deselected from the list. I will try to use another method (willStart() for example) that is only called once.

Reply:
Ok, that'll be better :+1:

Reply:
Done.